src/components/upload-form/upload-slide.js

```diff
@@ -133,7 +149,8 @@ class UploadSlide extends Component {
           <div className="upload-column">
             <p>Cost</p>
             <h3 className="storage-fees">
-              3 Gb for 10 years: <span> 30 PRL</span>
+              {this.state.fileSize / BYTES_IN_GIGABYTE} GB for {retentionYears} years:
+              <span> {this.state.storageCost} PRL</span>
```

Comment:
Is there any kind of rounding yet on these values?

Reply:
I mentioned that to edmund, but neither of us have heard anything about rounding recommendations. I think rounding the file size is a good idea, you think we should do the price?

We could round on the upload page and then show the full price on the invoice page.

Reply:
I agree with your recommendations.  Round the file size.  Truncate the price on the upload page and show the full price on the invoice page.